clients/java/src/main/java/io/camunda/client/CamundaClient.java

```diff
@@ -1885,6 +1889,103 @@ DecisionRequirementsGetXmlRequest newDecisionRequirementsGetXmlRequest(
    */
   VariableGetRequest newVariableGetRequest(long variableKey);
 
+  /**
+   * Creates a request to create a new cluster variable.
+   *
+   * <p>Cluster variables can be created with either global or tenant scope: /** Creates a request
+   * to create a new cluster variable.
+   *
+   * <p>Cluster variables can be created with either global or tenant scope:
+   *
+   * <pre>
+   *   camundaClient
+   *       .newClusterVariableCreateRequest()
+   *       .atGlobalScoped()
+   *       .create("myVariable", "myValue")
+   *       .send();
+   *
+   *   // or for tenant-scoped variable
+   *   camundaClient
+   *       .newClusterVariableCreateRequest()
+   *       .atTenantScoped("my-tenant-id")
+   *       .create("myVariable", "myValue")
+   *       .send();
+   * </pre>
+   *
+   * @return a builder for creating a cluster variable
+   */
+  ClusterVariableCreationCommandStep1 newClusterVariableCreateRequest();
```

Comment:
💭 The client DSL and API deviate here. In the API, we have different paths. In the client, we use different attributes. That can be a hint at misaligned concepts in the API (or client). If we think, as a Java client user, it's better UX to use the attribute way, the API should work the same way, potentially. The Java client way of reflecting the different paths would be creating different commands/requests, actually. So, `newGlobalScopeClusterVariableCreateRequest` and `newTenantScopeClusterVariableCreateRequest`, for example. If we think that's a bad idea/UX, it might be a bad idea/UX for the REST API as well 😅

@megglos, FYI, for awareness and maybe input here regarding API and client UX alignment.